azure/scope/managedcontrolplane.go

```diff
@@ -289,6 +296,19 @@ func (s *ManagedControlPlaneScope) VNetSpec() azure.ResourceSpecGetter {
 	}
 }
 
+// AzureFleetsMemberSpec returns the fleet spec.
+func (s *ManagedControlPlaneScope) AzureFleetsMemberSpec() azure.ASOResourceSpecGetter[*asocontainerservicev1preview.FleetsMember] {
+	return &fleetsmember.AzureFleetsMemberSpec{
+		Name:                 s.ClusterName(),
+		Namespace:            s.Cluster.Namespace,
+		ResourceGroup:        s.ResourceGroup(),
+		Group:                s.AzureFleetManager().Group,
+		SubscriptionID:       s.SubscriptionID(),
+		ManagerName:          s.AzureFleetManager().ManagerName,
```

Comment:
ok, I think I understand why this is so confusing.

if you rename the func AzureFleetmanager to 
`FleetMembership()` that will make more sense IMO.